worker_node/job/confirm.js

```diff
@@ -27,18 +28,15 @@ class Confirm extends Base {
     const transactionList = [];
     let timeoutTxCount = 0;
     let totalConfirmedTime = 0;
-    const confirmedTimeTable = {}; // TODO(csh): Delete after test
 
     for (let number = from; number <= to; number++) {
       const block = await this.#ain.getBlock(number, true);
+      this.output.statistics.transactionCount += block.transactions.length;
+      if (!this.config.saveTxs) {
```

Comment:
Could you add the saveTxs to sample config?

Reply:
Done! Thanks